docs/src/refs/api.md

```diff
@@ -37,16 +37,25 @@ GreedyScheduler
 SerialScheduler
 ```
 
-## Non-Exported
+## Re-exported
+
+|                        |                                                                     |
+|------------------------|---------------------------------------------------------------------|
+| `OhMyThreads.chunks`   | see [ChunkSplitters.jl](https://juliafolds2.github.io/ChunkSplitters.jl/stable/references/#ChunkSplitters.chunks) |
+| `OhMyThreads.index_chunks`   | see [ChunkSplitters.jl](https://juliafolds2.github.io/ChunkSplitters.jl/stable/references/#ChunkSplitters.index_chunks) |
```

Comment:
(FYI, you can link directly to the correct version with https://juliadocs.org/DocumenterInterLinks.jl/stable/)

Reply:
TIL. I'll try this in a different PR.